File: src/main/kotlin/nl/info/zac/search/model/DatumRange.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2024 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.zac.search.model

import java.time.LocalDate

data class DatumRange(val van: LocalDate?, val tot: LocalDate?)

File: src/main/app/src/app/zoeken/model/datum-range.ts
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

export class DatumRange {
  van: Date | null = null;
  tot: Date | null = null;

  constructor(van?: Date | null, tot?: Date | null) {
    this.van = van || null;
    this.tot = tot || null;
  }

  hasValue() {
    return this.van != null || this.tot != null;
  }
}
